app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -143,36 +157,10 @@ def label_for(file)
           end
         end
 
-        # Takes an optional block and executes the block if the save was successful.
-        # @return [Boolean] false if the save was unsuccessful
-        def save
```

Comment:
AFAIK, this code is nearly five years old to the day: https://github.com/samvera/sufia/commit/530868950a314e4dfdd78f2f02c262636bc70129#diff-47b795ef43d49d2d98f2e4a700c306b5R163

I'm not sure we still need it.